tasks/eth/031-uni-holocene-systemconfig-update/VALIDATION.md

```diff
@@ -0,0 +1,108 @@
+- Chain Governor simulation: `0xf45c50f658f1c1156ebc02208b87c846b9f5c470124d1bd92ce765f4c696abb2`
+
+calculated as explained in the nested validation doc:
+```sh
+SAFE_HASH=0x41f0e95f184f6402520b1628e6d1f8fe9153d45d5d4b66a3b768145563f90027 # "Nested hash:"
+SAFE_ROLE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03 # Council
+cast index bytes32 $SAFE_HASH $(cast index address $SAFE_ROLE 8)
+# 0x568b8aba0ab4ee2af4639a4fd31bba2c11adcebaec482afc2699d34c3b21f32b
+
+SAFE_ROLE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92 # Foundation
+cast index bytes32 $SAFE_HASH $(cast index address $SAFE_ROLE 8)
+# 0xb4a9e4ff89a9d93c76d7d832cbc444bc908574ac3e65bb031236999af8e42b74
+
+SAFE_ROLE=0xb0c4C487C5cf6d67807Bc2008c66fa7e2cE744EC # Unichain
+cast index bytes32 $SAFE_HASH $(cast index address $SAFE_ROLE 8)
+# 0xf45c50f658f1c1156ebc02208b87c846b9f5c470124d1bd92ce765f4c696abb2
+```
+
+## Safe Nonce Overrides
+
+### `0x847B5c174615B1B7fDF770882256e2D3E95b9D92` (Foundation `GnosisSafeProxy`)
+
+Only during Foundation simulation.
+
+We're overriding the nonce to increment the current value `12` by `1` to account for task `ink-001`.
+The simulation will also print out
+
+```
+Overriding nonce for safe 0x847B5c174615B1B7fDF770882256e2D3E95b9D92: 12 -> 13
+```
+And the state changes then show that this nonce is increased from 12 to 13 during the simulation.
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000005`<br/>
+  **Value:** `0x000000000000000000000000000000000000000000000000000000000000000d` (`13`)<br/>
+  **Meaning:** The Foundation Safe nonce is bumped from 12 to 13.
+
+The other state overrides are explained in the generic nested Safe validation document referred to
+above.
+
+### `0xc2819DC788505Aac350142A7A707BF9D03E3Bd03` (Security Council `GnosisSafeProxy`)
+
+Only during Security Council simulation.
+
+We're overriding the nonce to increment the current value `9` by `1` to account for task `ink-001`.
+The simulation will also print out
+```
+Overriding nonce for safe 0xc2819DC788505Aac350142A7A707BF9D03E3Bd03: 9 -> 10
+```
+And the state changes then show that this nonce is increased from 9 to 10 during the simulation.
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000005`<br/>
+  **Value:** `0x000000000000000000000000000000000000000000000000000000000000000a` (`10`)<br/>
+  **Meaning:** The Council Safe nonce is bumped from 9 to 10.
+
+The other state overrides are explained in the generic nested Safe validation document referred to
+above.
+
+## State Changes
+
+### `0xc407398d063f942feBbcC6F80a156b47F3f1BDA6` (`SystemConfigProxy`)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000066`
+  **Before**: `0x00000000000000000000000000000000000000000000000000000000000dbba0`
+  **After**: `0x010000000000000000000000000000000000000000000000000dbba0000007d0`
+  **Meaning**: Updates the `SystemConfig` proxy implementation.
```

Comment:
This is actually the scalar slot. See e.g. https://github.com/ethereum-optimism/superchain-ops/blob/ea0cadfbbc49c94d13202017c9ff2111d55cdd65/tasks/eth/028-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md?plain=1#L67-L70.

The `dbba0` bit being nonzero is what is causing the validation to fail.